android/README.md

```diff
@@ -0,0 +1,51 @@
+
+## Android Port of Foundation module
+
+This directory contains scripts used in the port of Foundation to
+an Android toolchain. The short version of the story is that after
+downloading the swift sources use the script "prepare.sh" in this
+directory to install prebuilt binaries and headers for libxml,
+libcurl and libdispatch then run the script "builder.sh".
+
+This build requires the path to a [r12 Android NDK](http://developer.android.com/ndk/downloads/index.html) in the
+`ANDROID_NDK_HOME` environment variable and the path to the
+android port of the "icu" libraries in `ANDROID_ICU_UC`
+downloaded from [here](https://github.com/SwiftAndroid/libiconv-libicu-android/releases/download/android-ndk-r12/libiconv-libicu-armeabi-v7a-ubuntu-15.10-ndk-r12.tar.gz).
+The port was tested against api 21 on a Android v5.1.1 LG K4 Phone (Lollipop)
+and requires an Ubuntu 15 host with the Android NDK Gold linker from
```

Comment:
So this won't build as a darwin host? that seems subpar; what would it take to fix that to actually build on a primary development platform of swift?